crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -36,8 +36,11 @@ use std::ops::Mul;
 #[derive(Component, Debug, PartialEq, Clone, Copy, Reflect)]
 #[reflect(Component, PartialEq)]
 pub struct GlobalTransform {
+    /// The position of the transform
```

Comment:
```suggestion
    /// The position of the global transform
```
I would prefer to avoid "transform" without global in this file (unless the doc is speaking about a transform) to reduce confusion between transforms and global transforms